### ISSUE TITLE
nice clear error message (adding agent with common name)-- still cant make progress

### ISSUE THREAD

**Issue by mkoo** [ArctosDB/arctos#4524]
**Describe the bug**
Trying to add in a new agent which has the exact same name as another clearly unrelated agent in Arctos. I cannot add it in by either force or trickery.

**To Reproduce**
1. Go to https://arctos.database.museum/agents.cfm?agent_id=21338980
2. Try to amend the name by removing in lastname the prefix 'notthesame'
3. See error below:
An error occurred and your changes were not saved.
In the event of multiple error messages, you may need to reload this page to continue. Save incrementally if necessary. 
ERROR: duplicate key value violates unique constraint "iu_agnt_preferred_name"
  Detail: Key (preferred_agent_name)=(James Bradley) already exists.:  SQL:UPDATE agent SET
						agent_remarks = 'graduate student, University of Chicago',
						agent_type='person',
						preferred_agent_name='James Bradley'
					WHERE
						agent_id = 21338980

I already set a relationship of "not the same as" for https://arctos.database.museum/info/agentActivity.cfm?agent_id=21251763

**Expected behavior**
I expect that I should be allowed to use the same preferred name with the relationship of "not the same as"! The unique constraint should be removed if we have a relationship as or other flag (perhaps a check box to allow exceptions). I mean they both have their own agent_id so they are not identical!

Whatever the solution, I can add documentation tips or explanation as needed.

**Comment by dustymc** [comment]
https://handbook.arctosdb.org/best_practices/Agents.html#different-agent-same-name

**Comment by mkoo** [comment]
Yea I went through that and I am saying that preferred name is not enough. we have other bits of agent details to distinguish between the two separate agent_ids. Parenthesis added to lastname is kinda too hacky and yucky as a workaround. (I know it is essentially what I did.)
I knew this was going to happen eventually. This is a common name combo and definitely not the same person. So creative solutions?
